study-builder/studies/pancan/study-pdfs.conf

```diff
@@ -0,0 +1,934 @@
+{
+  "pdfs": [
+    {
+      "name": "countmein-consent",
+      "filename": "countmein-consent",
+      "displayName": "countmein consent pdf",
+      "mapping": {
+        "type": ${id.act.consent},
```

Comment:
Although the string is the same, I'm not sure we should use the activity code here where we expect a mapping type?

Reply:
corrected.
Do we need this mapping !
Looks like I didn't add for other parental consent and also for release. Wondering if they are needed because I see pdfs are generated fine.